### PR TITLE
[VCDA-4161] Update the examples and capiYaml template in the Github repo

### PR DESCRIPTION
**Pull request by ymo24**
Signed-off-by: ymo24 <ymo@vmware.com>

## Description
Please provide a brief description of the changes proposed in this Pull Request

-  [Mustdo-GA]Update the examples and capiYaml template in the Github repo

## Checklist
- [X] tested locally
- [ ] updated any relevant dependencies
- [ ] updated any relevant documentation or examples

## API Changes
Are there API changes?
- [X] Yes
- [x] No

If yes, please fill in the below

1. Updated conversions?
    - [ ] Yes
    - [X] No
    - [ ] N/A
2. Updated CRDs?
    - [ ] Yes
    - [X] No
    - [ ] N/A
3. Updated infrastructure-components.yaml?
    - [ ] Yes
    - [X] No
    - [ ] N/A
4. Updated `./examples/capi-quickstart.yaml`?
    - [X] Yes
    - [ ] No
    - [ ] N/A
5. Updated necessary files under `./infrastructure-vcd/v1.0.0/`?
   - [X] Yes
   - [ ] No
   - [ ] N/A

## Issue
If applicable, please reference the relevant issue

Fixes #

- - -
This change is [Reviewable](https://reviewable.io/reviews/vmware/cluster-api-provider-cloud-director/225)

### CODE REVIEW

File: infrastructure-vcd/v1.0.0/cluster-template.yaml [hunk 1]
```diff
@@ -56,6 +56,8 @@ spec:
     k8sStorageClassName: ${K8s_STORAGE_CLASS_NAME} # name of the storage class in kubernetes
     useDeleteReclaimPolicy: ${USE_DELETE_RECLAIM_POLICY} # if set to true, Delete will be used as the Reclaim policy for the storage class. Else, Retain will be used as the reclaim policy
     fileSystem: ${FILE_SYSTEM} # file system format for the persistent volumes created from the storage class
+  loadBalancer:
+    vipSubnet: ${IP_SUBNET} # Virtual IP CIDR for the external network
```
Comment (Anirudh9794) [inline infrastructure-vcd/v1.0.0/cluster-template.yaml:60]: Please make sure IP_SUBNET is part of infrastructure-vcd/v1.0.0/clusterctl.yaml

**Review by Anirudh9794 — APPROVED** [review]

**Review by lzichong — APPROVED** [review]
:lgtm: after addressing Aniruddh's comment

> *[Reviewable](https://reviewable.io/reviews/vmware/cluster-api-provider-cloud-director/225#-:-N9goLCZBpHHWPgI4eU4:bhyemr)* status: 0 of 3 files reviewed, 1 unresolved discussion (waiting on @abaruni and @ymo24)

File: infrastructure-vcd/v1.0.0/cluster-template.yaml (hunk 1 above)
Comment (ymo24) [reply infrastructure-vcd/v1.0.0/cluster-template.yaml:60]: Hi Aniruddha, I have uploaded the changes. Thanks a lot

**Review by abaruni — APPROVED** [review]
> Reviewed 3 of 3 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/vmware/cluster-api-provider-cloud-director/225)* status: all files reviewed, 1 unresolved discussion (waiting on @Anirudh9794)

**Pull request merged**